src/backend/cc.h

```diff
@@ -195,6 +195,7 @@ struct Srcpos
     struct Sfile **Sfilptr;     // file
     #define srcpos_sfile(p)     (**(p).Sfilptr)
     #define srcpos_name(p)      (srcpos_sfile(p).SFname)
+    short Sfilnum;              // file number
```

Comment:
The improvement is the `Sfilnum` field is not needed by dmd, and it bumps up the size of `Srcpos`.